py-polars/polars/expr/expr.py

```diff
@@ -247,6 +247,7 @@ def to_physical(self) -> Self:
         - :func:`polars.datatypes.Time` -> :func:`polars.datatypes.Int64`
         - :func:`polars.datatypes.Duration` -> :func:`polars.datatypes.Int64`
         - :func:`polars.datatypes.Categorical` -> :func:`polars.datatypes.UInt32`
+        - ``List(Categorical)`` -> ``List(UInt32)``
```

Comment:
This should be physical for all nested types. I think we should mention that all the inner type is represented by the physical types stated above.